Samples/src/ReversibleLogicSynthesis/ReversibleLogicSynthesis.qs

```diff
@@ -122,17 +122,17 @@ namespace Microsoft.Quantum.Samples.ReversibleLogicSynthesis {
     /// Numbers(5); // [0, 1, 2, 3, 4]
     /// ```
     function Numbers (count : Int) : Int[] {
-        
+
         mutable array = new Int[count];
-        
+
         for (i in 0 .. count - 1) {
             set array w/= i <- i;
```

Comment:
Same here - essentially I'd like to get rid of these, because we should have convenient infrastructure in the array module.